industry_benchmarks/input_structures/prepared_structures/merck/shp2/ligands.sdf

```diff
@@ -0,0 +1,3215 @@
+SHP099-1/Example 7
```

Comment:
```suggestion
SHP099-1 Example 7
```
The `/` is going to cause some issues at simulation time (since it'll get added to the file name).